package.json

```diff
@@ -73,11 +75,9 @@
     "@readme/variable": "^13.4.12",
     "@semantic-release/changelog": "^5.0.1",
     "@semantic-release/git": "^9.0.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^10.1.0",
     "babel-jest": "^27.0.5",
     "babel-loader": "^8.2.2",
-    "babel-polyfill": "^6.26.0",
+    "chokidar": "^3.5.2",
```

Comment:
Where are you using Chokidar?

Reply:
It's a dep of webpack-dev-server. I **was** getting a `npm i` error, but it seems fine now.